src/eko/couplings.py

```diff
@@ -90,6 +92,8 @@ def expanded_nlo(ref, beta0, beta1, lmu):
 def expanded_nnlo(ref, beta0, beta1, beta2, lmu):
     """Compute expanded solution at NNLO.
 
+    From apfel
```

Comment:
please mention that this is *not* the default expression, but instead is coming from |APFEL| eq. ...

Reply:
Do you know where I can find a reference that I can cite?? In the Apfel paper and documentation i couldn't find anything

Reply:
mmm you're right - that makes me worry that it seems there is official statement ... when we first came across this (meaning implementing NNLO) we asked to the whole collaboration how to obtain this, but we only got alternative solutions ... I can see two options: or we trust APFEL and just cite it (without specifying a specific equation), or we fall back to the ordinary, iterated solution

Reply:
In lack of better references, cite APFEL